### ISSUE TITLE
In Public: when the Publisher views an asset the Subscription Options from Local Storage are inserted

### ISSUE THREAD

**Issue by kitblake** [chaineresearch/datastreams#31]
Screenshots:
![ConsumerStreamingAsset](https://user-images.githubusercontent.com/912545/212560094-e344208d-9211-4e1a-9980-e9a539b0bf05.png)
First StreamingAsset
  \
  \
<img width="415" alt="PublisherStreamingAsset" src="https://user-images.githubusercontent.com/912545/212560118-9252627e-c7bf-474a-9f90-fc7dedc6323e.png">
Same StreamingAsset after publishing a different one

**Comment by kitblake** [comment]
To fix this let's just turn off Local Storage. It means that a Publisher could reload and lose the values filled into the form fields, but that's not a disaster. In a later phase we can implement this optimization. 

This will probably also fix Issue #30.

**Comment by kitblake** [comment]
To solve this requires two steps:
1. The pricing metadata (Subscription Options) needs to be stored in the DDO, e.g. something like this row:
  ```
  "subsPrice": {
      "rows": [
        {
          "unit": 4,
          "time": "days",
          "price": 1000,
          "isNew": false
        }
      ]
    },
  ```
2. When rendering the public view of an asset the Subscription Options need to be retrieved from the DDO.

We’ve done this before.
